edgePy/DGEList.py

```diff
@@ -353,22 +365,40 @@ def create_DGEList(
         )
 
     @classmethod
-    def create_DGEList_handle(
-        cls, data_handle: StringIO, group_file: Path, **kwargs: Mapping
+    def create_DGEList_data_file(
+        cls, data_file: Path, group_file: Path, **kwargs: Mapping
     ) -> "DGEList":
         """Read in a file-like object of delimited data for instantiation.
 
         Args:
-            data_handle: Any handle supporting text streaming io.
+            data_file: Text File defining the data set.
             group_file: the json file defining the groups
```

Comment:
`the` should probably be uppercase. Otherwise docstrings look great!

Reply:
fixed.